examples/speaker_recognition/spkr_get_emb.py

```diff
@@ -49,12 +39,15 @@
 def main(cfg):
 
     logging.info(f'Hydra config: {cfg.pretty()}')
-    trainer = pl.Trainer(logger=False, checkpoint_callback=False)
+    if cfg.trainer.gpus > 1:
```

Comment:
Wait do this only during inference (trainer.test()) otherwise you can't use multi GPU training

Reply:
spkr_get_emb.py is only run for inference purposes.

Reply:
Ah ok.